unison-src/transcripts/reflog.output.md

```diff
@@ -44,55 +43,93 @@ y = 2
 
 ```
 ```ucm
-.> add
+scratch/main> add
 
   ⍟ I've added these definitions:
   
     y : Nat
 
-.> view y
+scratch/main> branch /other
 
-  y : Nat
-  y = 2
+  Done. I've created the other branch based off of main.
+  
+  Tip: To merge your work back into the main branch, first
+       `switch /main` then `merge /other`.
+
+scratch/other> alias.term y z
+
+  Done.
+
+newproject/main> builtins.merge lib.builtins
+
+  Done.
+
+newproject/main> alias.type lib.builtins.Nat MyNat
+
+  Done.
 
 ```
+Should see reflog entries from the current branch
+
 ```ucm
-.> reflog
+scratch/main> reflog
```

Comment:
check if reflog accepts another branch, or add a feature request for it

Reply:
<img width="717" alt="image" src="https://github.com/unisonweb/unison/assets/538571/87f3e8ed-5fb4-4239-aa86-4bc719526f61">